directory_path.cpp

```diff
@@ -28,6 +29,15 @@ bool DirectoryPath::exists() const {
   return isDirectory(getPath());
 }
 
+bool DirectoryPath::make() {
```

Comment:
Doesn't this have the same functionality as the function below it? You only call it with 'if (!to.exists()) to.make();'

Reply:
Yes, I didn't notice it :)